o4/o4.py

```diff
@@ -1166,8 +1171,8 @@ def o4_status(changelist, depot, check_all, quick):
         print("")
         print(f"*** INFO: Besides the {len(has_open)} file{s} opened for edit, "
               f"all files passed the checksum test.")
-
-    return True
+        return 0
+    return len(all_fnames)
```

Comment:
Should be length of naughty I think. There's no error just because you have files open for edit.